### ISSUE TITLE
`repeat search with omitted results included` link is broken

### ISSUE THREAD

**Issue by deepseagirl** [deepseagirl/goo-gone#1]
![image](https://user-images.githubusercontent.com/47490856/129653755-159f4474-32fa-4df8-a6df-a9ab12e2ea6f.png)

intended behavior: "include omitted" link adds `filter=0` parameter to the url of the `google.com/search?...` page you are already on, causing omitted results to be included.

current behavior: "include omitted" link is malformed due to over matching of goo-gone filter: `https://www.google.com/url?q=/search%3Fq%3D...`

tldr that link should be left alone.
